pkg/kubeapiserver/options/admission.go

```diff
@@ -116,8 +116,13 @@ func (a *AdmissionOptions) ApplyTo(
 
 	if a.PluginNames != nil {
 		// pass PluginNames to generic AdmissionOptions
-		a.GenericAdmission.EnablePlugins = a.PluginNames
+		a.GenericAdmission.EnablePlugins, a.GenericAdmission.DisablePlugins = computePluginNames(a.PluginNames, a.GenericAdmission.RecommendedPluginOrder)
```

Comment:
You calculate disabled plugins here.Originally, I do this in generic apiserver. Both works well.